CONTRIBUTING.md

```diff
@@ -21,6 +21,12 @@ The PR comment should be formed by a one-line subject, followed by one line of w
 
 Where your PR fixes an issue, it should include a reference to the issue's identifier in the first line of the commit comment.  The comment should provide enough information for a reviewer to understand the changes and their relation to the rest of the code.
 
+### Running Tests
+There are labels that can be added to this pull request to trigger GitHub Actions builds:
+- `build-linux` - trigger all github actions Linux builds - RECOMMENDED
+- `build-windows` - trigger all github actions Windows builds
+- `build-macos` - trigger all github actions macOS builds - please use sparingly
+
```

Comment:
This may be better placed in the [prTester/README.md](https://github.com/AdoptOpenJDK/openjdk-build/tree/master/pipelines/build/prTester) doc, with a note placed here or in the FAQ that links to said doc.